### ISSUE TITLE
Add map to web app

### ISSUE THREAD

**Issue by dellisd** [cuhacking/atlas#17]
Add a map that fills the entire screen centered around the Carleton University campus. This should be implemented through React and in TypeScript.

Add a layer with a random feature visible on the map.

This map will be used to test implementations of Mapbox classes and functions in future tasks.

This task is very similar to cuhacking/atlas-editor#1, but the difference will be where the Mapbox access token comes from. In that task, it came from a `.env` file, but here it will come from the compiled Kotlin/JS code.

# TypeScript Interop

Before starting, make sure to follow the build instructions in the README. You will need to make sure you run `gradlew build` otherwise the compiled JavaScript won't be visible and yarn will complain.

Because we are using TypeScript, we need TypeScript definitions for the compiled Kotlin/JS code. In Kotlin 1.4 the compiler will automatically generate the definitions for us, but we're stuck on 1.3.70 for the time being... so we have to write them manually. This was already done in 4016cb4, but nonetheless the way you import these definitions is a little strange since Kotlin namespaces all of the compiled code.

You can see this in `index.tsx` under the web directory, like this:

```ts
import { com } from "Atlas-common";
import BuildKonfig = com.cuhacking.atlas.common.BuildKonfig;

// Use it here
BuildKonfig.MAPBOX_KEY
```

This constant was compiled from Kotlin code, how cool is that??
